GETTING-STARTED.md

```diff
@@ -1,281 +0,0 @@
-# PatternFly React Getting Started
```

Comment:
Where are we capturing some of this information that is still relevant?

Reply:
Which parts aren't covered?